jormungandr-lib/src/interfaces/block0_configuration/BLOCKCHAIN_CONFIGURATION_DOCUMENTED_EXAMPLE.yaml

```diff
@@ -215,3 +217,18 @@ initial:
   #    # Legacy Cardano address
   #    - address: 48mDfYyQn21iyEPzCfkATEHTwZBcZJqXhRJezmswfvc6Ne89u1axXsiazmgd7SwT8VbafbVnCvyXhBSMhSkPiCezMkqHC4dmxRahRC86SknFu6JF6hwSg8
   #      value: 123
+
+# Initial token distribution
+initial_tokens:
```

Comment:
Why separate section?

Reply:
It was added as a additional field inside the `Block0Configuration` for the purpose do not break the current invariant and usage of the `Initial` struct - "one `Initial` == one `Fragment`"

Reply:
updated